types/jasmine-data_driven_tests/jasmine-data_driven_tests-tests.ts

```diff
@@ -51,3 +51,10 @@ describe("A suite", () => {
         }
     );
 });
+
+all("A data set must consist of array-wrapped arrays, if test expects single array input",
```

Comment:
I'd rename this to something like `A data driven test can have array arguments` to be consistent with other tests and move it next to the other `all` tests, but looks good to me.

Reply:
my intention was to indicate that it is necessary to wrap array arguments in additional arrays, since otherwise it simply won't work